### ISSUE TITLE
[question] Can user reset the seed of a dataloader?

### ISSUE THREAD

**Issue by Freed-Wu** [NVIDIA/DALI#4239]
I notice <https://github.com/NVIDIA/DALI/blob/8b8e7c6521c4de78fb6774fdf3263d6ded47a6df/docs/examples/use_cases/pytorch/resnet50/main.py#L355>

```python
train_loader.reset()
```

Does any method which allow user to reset the seed exists? Like:

```python
seed = XXX
train_loader.reset(seed=seed)
```

Thanks!

**Comment by JanuszL** [comment]
Hi @Freed-Wu,

The seed is given once for the pipeline lifetime. If you want to change it you need to recreate the pipeline. The main reason is the asynchronous execution model of DALI. Even if you call `train_loader.reset() ` resets the python wrapper state over the native pipeline which may already compute batch for the next iteration (not to mention the state on the GPU that is also asynchronous compared to the CPU).
